slo-monitor/src/monitors/watcher.go

```diff
@@ -28,7 +28,6 @@ import (
 
 const (
 	resyncPeriod      = time.Duration(0)
-	watchListPageSize = 5000
```

Comment:
@mborsz - given we don't need this, do we want to revert the changes you were doing?

Reply:
Yes, once we validate this approach, we can submit follow up PR that will remove this, fork of client-go and other ugly stuff I added.